### ISSUE TITLE
Using the scrollbar of the parent page selector closes the selector.

### ISSUE THREAD

**Issue by jennydupuy** [WordPress/gutenberg#27283]
### Description
When i'm using the scrollbar in the parent page selector, the selector closes itself.
The bug is occuring under Chrome and Firefox.

### To reproduce
1.Go to the page editor
2.Activate the parent page selector
3.Click on scrollbar to scroll down or up : the selector closes itself

### Expected behavior
Scroll up or down with the scrollbar without the selector closes itself.

### Screenshots
![with-scrollbar](https://user-images.githubusercontent.com/75045144/100286954-0fa33c80-2f74-11eb-8d07-2ec8272ea631.gif)

**Editor version (please complete the following information):**
- WordPress version: 5.5.3
- The bug is present from version 9.2 of Gutenberg, whether the plugin is installed or not.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser  Chrome 87.0.4280.66 (Build officiel) (64 bits)
 - Browser Firefox 83.0 (64 bits)

**Comment by talldan** [comment]
This is also reproducible in the storybook example for the combobox react component:
https://wordpress.github.io/gutenberg/?path=/story/components-comboboxcontrol--default

So it should be that one component that's the source of the issue.
